client/jetpack-cloud/sections/partner-portal/primary/payment-method-list/index.tsx

```diff
@@ -12,13 +13,112 @@ import Main from 'calypso/components/main';
+import EllipsisMenu from 'calypso/components/ellipsis-menu';
+import PopoverMenuItem from 'calypso/components/popover/menu-item';
+import Gridicon from 'calypso/components/gridicon';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+function StoredCard( props ): ReactElement {
+	let logo;
+	const translate = useTranslate();
+
+	if ( 'visa' === props.card.card_type ) {
+		logo = <VisaLogo />;
+	} else if ( 'mastercard' === props.card.card_type ) {
+		logo = <MastercardLogo />;
+	} else if ( 'amex' === props.card.card_type ) {
+		logo = <AmexLogo />;
+	}
+
+	const expiry = new Date( props.card.expiry );
+	const month = expiry.getMonth();
+	const year = expiry.getFullYear().toString().substr( 2, 2 );
+
+	return (
+		<div className="payment-method-list__stored-card">
+			<div className="payment-method-list__stored-card-header">
+				<div className="payment-method-list__stored-card-header-left">{ logo }</div>
+
+				<div className="payment-method-list__stored-card-header-right">
+					<Gridicon icon="checkmark" size={ 18 } />
+					<div className="payment-method-list__stored-card-primary">
+						<span className="payment-method-list__stored-card-primary-text">
+							{ translate( 'Primary' ) }
+						</span>
+					</div>
+					<EllipsisMenu className="payment-method-list__stored-card-ellipsis" position="bottom">
+						<PopoverMenuItem>{ translate( 'Make primary' ) }</PopoverMenuItem>
+						<PopoverMenuItem>{ translate( 'Update details' ) }</PopoverMenuItem>
+						<PopoverMenuItem>{ translate( 'Delete card' ) }</PopoverMenuItem>
+					</EllipsisMenu>
+				</div>
+			</div>
+			<div className="payment-method-list__stored-card-footer">
+				<div className="payment-method-list__stored-card-footer-left">
+					<div className="payment-method-list__stored-card-name">{ props.card.name }</div>
+					<div className="payment-method-list__stored-card-number">
+						**** **** **** { props.card.card }
+					</div>
+				</div>
+
+				<div className="payment-method-list__stored-card-footer-right">
+					<div className="payment-method-list__stored-card-expiry">{ `${ month }/${ year }` }</div>
+				</div>
+			</div>
+		</div>
+	);
+}
+
+function AddStoredCard(): ReactElement {
+	const translate = useTranslate();
+	const dispatch = useDispatch();
+
+	const navigateToCreateMethod = () => {
+		dispatch(
+			recordTracksEvent( 'calypso_partner_portal_license_list_empty_issue_license_click' )
+		);
+
+		page( '/partner-portal/payment-method/add' );
+	};
+
+	return (
+		<div
+			className="payment-method-list__add-card"
+			onClick={ navigateToCreateMethod }
+			onKeyDown={ navigateToCreateMethod }
+			role="button"
+			tabIndex={ 0 }
+		>
+			<div className="payment-method-list__add-card-content">
+				<CardHeading className="payment-method-list__add-card-text" size={ 18 }>
+					{ translate( 'Add new credit card' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **23** times:
`translate( 'Add a new credit card' )` *ES Score: 9*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/jetpack-partner-portal-payment-method-list)